lottie/src/main/java/com/airbnb/lottie/LottieFeatureFlags.java

```diff
@@ -0,0 +1,48 @@
+package com.airbnb.lottie;
+
+import android.annotation.SuppressLint;
+import android.os.Build;
+
+import com.airbnb.lottie.utils.Logger;
+
+import java.util.HashSet;
+
+public class LottieFeatureFlags {
```

Comment:
Let's move `LottieFeatureFlags` to a package that's package-private so we don't expose this class in the public API and make `FeatureFlag` a top-level public enum `LottieFeatureFlag`